### ISSUE TITLE
Keyboard users cannot access "Connect to..." and "Shell" controls in Query Playground editor:A11y_DocumentDB for VS Code Extension_New Query Playground_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-documentdb#773]
## GitHub Tags
#A11yTCS; #E2E_DocumentDB_Win32_JUNE2026; #DocumentDB for VS Code Extension; #DesktopApp; #FTP; #Win32; #A11ySev2; #A11yMAS; #WCAG2.1.1; #Keyboard;  

## Issue Subcategory 
Element not reachable via keyboard 

## User Experience 
Keyboard-only users (including screen reader users and those with motor disabilities) cannot access or interact with the control at all. This creates a complete barrier to that functionality.

## Repro Steps 
Open VS code > Install the DocumentDB for VS Code Extension and open it > Add new Connections > Expand one of the databases > Expand a collection within that database > Open context menu for restaurants collection tree item > New Query Playground menu item > "Connect to....." or Shell button > Observe the issue: Keyboard users cannot access "Connect to..." and "Shell" controls in Query Playground editor.

## Actual Result 
Keyboard focus does not move to the "Connect to..." and "Shell" controls. These controls can be activated using a mouse, but keyboard users cannot navigate to or interact with them.

## Expected Result 
The "Connect to..." and "Shell" controls should be reachable and operable using the keyboard. Users should be able to access all functionality provided by these controls without requiring a mouse. The controls should receive visible keyboard focus and support standard keyboard interaction.

## How to Fix 
Use natively focusable elements (button, a href, input). For custom controls, add tabindex="0" and implement keyboard event handlers for activation. 

## Environment details 
Application Name: VS Code 
Extension: DocumentDB for VS Code
Extension Version: 0.9.1
Edition Windows 11 Enterprise
Version 25H2
OS build 26200.8390

## Note

- Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
- "[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachment

https://github.com/user-attachments/assets/2ea3f84c-c508-4fb9-894c-681e4e76815e

**Comment by tnaum-ms** [comment]
Got it. It's a general issue with the code lens approach we use here. We'll revisit.

**Comment by Mana-swi** [comment]
#a11ydb-keyboard-unreachable;
